pages/about/groups/task-forces/wcag2ict/work-statement.md

```diff
@@ -24,113 +24,127 @@ The WCAG2ICT Task Force is a task force of the [Accessibility Guidelines Working
+    *   provide a concise, high-level overview or introduction to WCAG2ICT, explaining its purpose and how to use it
+    *   where known, give examples of some Success Criteria which may not be applicable to a particular non-web technology
+    *   as needed, suggested changes to success criteria normative language to more clearly align with non-web software and document technologies
+    *   how each of the WCAG 2.x Level AAA success criteria could apply to non-web technologies
+
+NOTE: "suggested changes to success criteria" would not be suggestions to change WCAG itself, but instead to provide language more meaningful in non-web contexts. For example, in criteria where the WCAG normative language uses CSS pixels, the Task Force could provide alternative language more appropriate for non-web technologies.
 
 Explicitly out of scope for the WCAG2ICT Task Force work are:
 
-*   proposing changes to WCAG 2.x;
-*   developing techniques or interpretations of WCAG 2.x for implementing WCAG 2.x in Web technologies;
-*   developing any specific, sufficient techniques, including platform-specific techniques, for implementing WCAG 2.x in non-Web ICT, though examples may be in scope;
-*   determining whether WCAG, as a whole, _should be_ applied to non-web content; and
-*   developing a normative standard or any normative requirements.
+*   proposing changes to WCAG 2.x
+*   developing techniques or interpretations of WCAG 2.x for implementing WCAG 2.x in Web technologies
+*   developing any specific, sufficient techniques, including platform-specific techniques, for implementing WCAG 2.x in non-Web ICT, though examples may be in scope
+*   determining whether WCAG, as a whole, _should be_ applied to non-web content
+*   developing a normative standard or any normative requirements
 
 ## Approach
 
-WCAG2ICT is a Task Force of the AG WG, which has oversight over the Task Force to ensure accurate interpretation and representation of WCAG 2.x. The AG WG decision process applies within the group. We have an AG WG co-chair liaison to ensure draft content is taken to the AG WG for review as it progresses.
+WCAG2ICT is a Task Force of the AG WG, which has oversight over the Task Force to ensure accurate interpretation and representation of WCAG 2.x. The AG WG decision process applies within the group. Draft content is periodically taken to the AG WG for review as it progresses.
 
 The primary effort of this Task Force is to describe how to interpret WCAG 2.x and its principles, guidelines, success criteria and conformance requirements when applying them to non-web ICT; and evaluating what WCAG Conformance means in the context of non-Web ICT. The resulting Working Group Note will include a discussion of the application of each of the criteria in WCAG 2.x to non-Web ICT, including their interface components and platforms, and the extent to which [WCAG Conformance](http://www.w3.org/TR/WCAG20/#conformance) is meaningful to non-Web ICT.
```

Comment:
> [WCAG Conformance](http://www.w3.org/TR/WCAG20/#conformance)

This link points to the "Conformance section" of WCAG **2.0**. Is this intentional?

Reply:
The content of WCAG conformance hasn't changed in any of the versions. However, I did update the link to go to WCAG 2.2 to ensure links are to the latest.

Reply:
Thanks @maryjom